docs/user-tutorial/benchmarks/micro-benchmarks.md

```diff
@@ -60,11 +60,40 @@ Large scale matmul operation using `torch.matmul` with one GPU.
 
 ### `cublas-function`
 
-TODO
+#### Introduction
+
+The common cublas functions with parameters in models (resnet, vgg, densenet, bert, gpt2, lstm) traning.
+
+Cublas functions support:
```

Comment:
The support functions for cuBLAS are as follows:

Reply:
Updated